### PR TITLE
Updated the first figure

### PR DESCRIPTION
**Pull request by hannestschofenig**
image

### CODE REVIEW

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -364,8 +363,10 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
     / Collect Receipts /      /   Replay Log   /
    '------------------'      '----------------'
 ~~~
+{: #fig-concept-relationship title="Relationship of Concepts in SCITT"}
 
-This section describes at a high level, the three main roles and associated processes in SCITT: Issuers and Signed Statements, Transparency Service and the Signed Statement Registration process, as well as Relying Parties of the Transparent Statements and the Receipt validation process.
+This section describes at a high level, the three main roles and associated processes in SCITT:
```
Comment (hannestschofenig) [inline draft-ietf-scitt-architecture.md:368]:
```suggestion
This section describes at a high level, the three main roles and associated processes in SCITT:
```

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -327,23 +327,22 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
...
  | Statement |            '---+--+----'
-  '----+----'       cose sign |  | cose verify
+  '----+----'                 |  |
        |    .----------------'|  |
        |   |                  |  |
        v   v                  |  |'----------.
   .----+---+---.              |  |            |
  |    Signed    |             |  |            |
  |   Statement  |             |  |            |
- | (COSE_Sign1) |             |  |            |
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:337]: This current document reflects shifting from using "envelope" to referencing COSE_Sign1. See #245 

Personally, I'd like to keep the current state for the diagram

File: draft-ietf-scitt-architecture.md [hunk 3]
```diff
@@ -364,8 +363,10 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
     / Collect Receipts /      /   Replay Log   /
    '------------------'      '----------------'
 ~~~
+{: #fig-concept-relationship title="Relationship of Concepts in SCITT"}
 
-This section describes at a high level, the three main roles and associated processes in SCITT: Issuers and Signed Statements, Transparency Service and the Signed Statement Registration process, as well as Relying Parties of the Transparent Statements and the Receipt validation process.
+This section describes at a high level, the three main roles and associated processes in SCITT:
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:368]: 👍 for this section, lines 366-369.
If we could separate these changes from the architecture changes above, I'd LGTM these lines

**Review by OR13 — APPROVED** [review]

File: draft-ietf-scitt-architecture.md (hunk 2 above)
Comment (henkbirkholz) [inline draft-ietf-scitt-architecture.md:337]: Not a fan, please keep for now.

**Review by henkbirkholz — DISMISSED** [review]

**Review by SteveLasker — CHANGES_REQUESTED** [review]
I'm supportive of [368](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/266/files#diff-585fb424519b79cf000445f3425fa56d328cbaca5d2a622740ddc4e5ca91dbe1R368) through 369, but request the diagram remain as it rolls back the conversation in #245

File: draft-ietf-scitt-architecture.md (hunk 2 above)
Comment (roywill) [reply draft-ietf-scitt-architecture.md:337]: Agreed

**Review by roywill — CHANGES_REQUESTED** [review]

**Comment by hannestschofenig** [comment]
My biggest problem with the diagram is that it is not explained. It just falls out of the sky

File: draft-ietf-scitt-architecture.md [hunk 4]
```diff
@@ -327,23 +327,22 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
        v                 |  Identity   |
   .----+----.            |  Documents  |
  | Statement |            '---+--+----'
-  '----+----'       cose sign |  | cose verify
+  '----+----'                 |  |
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:330]:
```suggestion
'----+----' cose sign | | cose verify
```

File: draft-ietf-scitt-architecture.md [hunk 5]
```diff
@@ -327,23 +327,22 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
        v                 |  Identity   |
   .----+----.            |  Documents  |
  | Statement |            '---+--+----'
-  '----+----'       cose sign |  | cose verify
+'----+----' cose sign | | cose verify
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:330]:
```suggestion
 '----+----' cose sign | | cose verify
```

File: draft-ietf-scitt-architecture.md (hunk 4 above)
Comment (henkbirkholz) [inline draft-ietf-scitt-architecture.md]:
```suggestion
  '----+----' cose sign | | cose verify
```

File: draft-ietf-scitt-architecture.md (hunk 2 above)
Comment (yogeshbdeshpande) [reply draft-ietf-scitt-architecture.md:337]: I agree with @SteveLasker : I would not overcomplicate the already cumbersome figure any more.

Lets drop this PR. We do not need it!

**Review by yogeshbdeshpande — DISMISSED** [review]
I think, the addition of `cose sign` and `cose verify` is superficial and not needed.

Please remove them!

**Review by aj-stein-nist — CHANGES_REQUESTED** [review]
Like @yogeshbdeshpande, I am willing to quickly approve if the labels are changed to simply `sign` and `verify`.

**Comment by SteveLasker** [comment]
I took on the task to fix-up the markdown, testing it locally, posting a picture, ...
I didn't get to it today, but will on Wednesday. 
...please stand by... :)

**Comment by SteveLasker** [comment]
> My biggest problem with the diagram is that it is not explained. It just falls out of the sky

Thanks, Hannes, that's a valid point. We may have lost some of the descriptive text along with the edits.
Let's keep the diagram as-is, and work on additional text if needed.

**Comment by SteveLasker** [comment]
Latest changes, incorporating the feedback to date, which includes some formatting changes to the diagram, and text updates.
image

**Review by SteveLasker — APPROVED** [review]
LGTM to https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/266/commits/be1962374523437eb6ebf0ceaf6317a87a88760c

**Comment by yogeshbdeshpande** [comment]
I think the diagram on the master/main is fine just needs some sentences of explanation text, around it.

Already the diagram is having quite a bit of components and flows. I would strictly request everyone to

1. STOP editing the diagram
2. Add text around it trying to explain the philosophy presented via a simple SCITT Architecture.

**Comment by SteveLasker** [comment]
Thanks, @yogeshbdeshpande, the diagram content has been reverted, only incorporating a few formatting changes. 
Hannes, and subsequent merged suggestions, added some additional text. Can you comment, suggest, and/or approve the current state of this PR. It does add more context.

**Comment by yogeshbdeshpande** [comment]
Thanks Steve, looking now!

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Comment by yogeshbdeshpande** [comment]
Done and Thanks again!

**Pull request merged**